nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -674,20 +661,11 @@ public void onPropertyModified(final PropertyDescriptor descriptor, final String
 
         ProxyConfiguration.validateProxySpec(validationContext, results, PROXY_SPECS);
 
-        for (String headerKey : validationContext.getProperties().values()) {
-            if (excludedHeaders.containsKey(headerKey)) {
-                // We're not using the header message format string here, just this
-                // static validation message string:
-                results.add(new ValidationResult.Builder().subject(headerKey).valid(false).explanation("Matches excluded HTTP header name").build());
-            }
-        }
-
```

Comment:
is it some dead code?

Reply:
This validation block was related to the `excludedHeaders` check.  However, the way this was written, it was not actually working properly since `getProperties().values()` returns property _values_ instead of property _names_.  If the check for `Trusted Hostname` is absolutely necessary, this could be corrected, but I recommend removing all references.